wikimedia/wikipedia-ios
# T376320: Part 1

**Phabricator:** 
https://phabricator.wikimedia.org/T376320

### Notes
* Grabs data, will send updated slides if available

### Test Steps
1. Using account that has edits on Italian/French, set yourself up to access the YIR entry point
2. Go through slides
3. You should see your amount of edits on slides (if you have any)

### Screenshots/Videos
